pandas/core/indexes/datetimes.py

```diff
@@ -913,6 +913,45 @@ def to_series(self, keep_tz=False):
                       index=self._shallow_copy(),
                       name=self.name)
 
+    def to_frame(self, index=True, keep_tz=False):
+        """
+        Create a DataFrame with the columns the levels of the Index
+
+        .. versionadded:: 0.21.0
+
+        Parameters
+        ----------
+        index : boolean, default True
+            return the DatetimeIndex as the index
```

Comment:
same

Reply:
Yeah...bad English copied and pasted.  Fixed.